clients/go/pkg/commands/delete_resource.go

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"context"
+	"github.com/camunda/zeebe/clients/go/v8/pkg/pb"
+)
+
+type DeleteResourceCommandStep1 interface {
+	ResourceKey(int64) DispatchDeleteResourceCommand
+}
+
+type DispatchDeleteResourceCommand interface {
+	Send(context.Context) (*pb.DeleteResourceResponse, error)
+}
+
+type DeleteResourceCommand struct {
+	Command
+	request pb.DeleteResourceRequest
+}
+
+func (cmd *DeleteResourceCommand) Send(ctx context.Context) (*pb.DeleteResourceResponse, error) {
```

Comment:
💭 I was suprised that we return the generated protobuf type here, and not an interface. But it looks like this is how it is done on other commands as well.

I know in other clients we try to hide all of the grpc/protobuf as much as possible so we can also easier replace this.